moto/dynamodb2/models.py

```diff
@@ -5,9 +5,11 @@
 import json
 import re
 
+import boto3
```

Comment:
nix this?

Reply:
Is used to grab a list of regions that dynamo is present in, I'd rather that than hardcoding regions. Is also part of the script that makes models.py templates.

Reply:
Ohh I see! You also did that in the batch and polly services. This makes sense now.